docs/core-concepts/app-deployment.md

```diff
@@ -1,17 +1,82 @@
-# 🔧 .NET Core Application Deployment concepts
-
-> **Note**
-> 
-> This topic hasn’t been written yet! 
->
-> We welcome your input to help shape the scope and approach. You can
-> track the status and provide input on this
-> [issue](https://github.com/dotnet/core-docs/issues/471) at GitHub.
-> 
-> If you would like to review early drafts and outlines of this
-> topic, please leave a note with your contact information in the issue.
->
-> Learn more about how you can contribute on
-> [GitHub](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md).
->
-	
\ No newline at end of file
+# .NET Core Application Deployment concepts
```

Comment:
Right you are.  Feel free to close this one and pick up any content you'd like to reuse.  I wrote this as part of the doc-day meeting where this one was showing up as unclaimed in a spreadsheet and I didn't see any content online.  I neglected to check PRs.